packages/hs-react-ui/jest.config.js

```diff
@@ -0,0 +1,4 @@
+const {defaults} = require('jest-config');
```

Comment:
If you got tests running, could you add the script to the `package.json` so we can run it from CircleCI?

Reply:
Also great job!

Reply:
Couldn't get the run to recognize the file when I ran it so I removed it for this PR and and going to work on it this weekend to see if I can get the setup correct